app/web/template/sign_your_lpa.gohtml

```diff
@@ -6,6 +6,52 @@
             <h1 class="govuk-heading-xl">
                 {{ tr .App "signYourLpa" }}
             </h1>
+
+            {{ trFormatHtml .App "signYourLpaContent" "AllAttorneysFullNames" .Lpa.AttorneysFullNames "AllAttorneysFirstNames" .Lpa.AttorneysFirstNames }}
+
+            <div class="govuk-warning-text">
+                <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                <strong class="govuk-warning-text__text">
+                    <span class="govuk-warning-text__assistive">{{ tr .App "warning" }}</span>
+                    {{ trFormat .App "attorneysCanActOnlyIfAskedWarning" "AllAttorneysFirstNames" .Lpa.AttorneysFirstNames }}
+                </strong>
+            </div>
+
+            {{ trFormatHtml .App "attorneysMustFollowMCA" }}
+
+            {{ template "details" (details . "whatHappensIfIWantToMakeChanges" "whatHappensIfIWantToMakeChangesContent" false) }}
+
+            <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+            {{ trHtml .App "formPanelContent" }}
+
+            <form novalidate method="post">
+                <article class="moj-ticket-panel" aria-label="Sub navigation 1">
+                    <section class="moj-ticket-panel__content moj-ticket-panel__content--blue" aria-label="Section 1">
```

Comment:
These `aria-labels` need removing (or setting to something sensible). But also, if a screen reader user gets a label then why wouldn't someone reading on a screen get one?

Reply:
Also these should be `<div>`s rather than `<article>` or `<section>`. It isn't an article, and a section would have a title.

Reply:
I missed updating the labeling here from a C&P from MOJ pattern library (also I'm not even sure what we would call this?). That's where the article and section elements were defined - https://design-patterns.service.justice.gov.uk/components/ticket-panel/. I'd want to understand why they went with `article` and `section` in the first place and if changing to `div`s affects any of the UR research on this before changing. I'll see if anyone from MOJ frontend is around still to query with.

Reply:
Agreed in the context of this declaration element `article` and `content` doesn't make semantic sense. We'll use `div`s instead and drop the `aria-label` as it's not directly linked to an interactive element.